packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx

```diff
@@ -371,8 +638,55 @@ const GivingSeasonHeader = ({
                   )}
                 </div>
               )}
-              {rightHeaderItems}
+              <div className={classes.gsRightHeaderItems}>
+                {rightHeaderItems}
+              </div>
             </Toolbar>
+            <div className={classes.gsContent}>
+              <div>
+                <span>Get your 2023 donations in</span>
+              </div>
+              <div className={classes.gsContentSubtitle}>
+                {currentUser
+                  ? (
+                    <span>
+                      Forum users who already donated: consider adding a heart
+                      by clicking on this banner.
+                    </span>
+                  )
+                  : (
+                    <span>
+                      Explore the funds below — or log in to add a heart if
+                      you’ve already donated.
+                    </span>
+                  )
+                }
+              </div>
+              <div className={classes.gsButtons}>
+                <EAButton href="https://funds.effectivealtruism.org/funds/animal-welfare">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  Animal Welfare Fund
+                </EAButton>
+                <EAButton href="https://funds.effectivealtruism.org/funds/far-future">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  Long-term Future Fund
+                </EAButton>
+                <EAButton href="https://funds.effectivealtruism.org/funds/global-development">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  Global Development Fund
+                </EAButton>
+                <EAButton href="https://funds.effectivealtruism.org/funds/ea-community">
+                  <span className={classes.gsButtonIcon}>{lightbulbIcon}</span>{" "}
+                  EA Infrastructure Fund
+                </EAButton>
+                <EAButton
+                  href="/posts/bBm64htDSKn3ZKiQ5/meet-the-candidates-in-the-forum-s-donation-election-2023"
+                  className={classes.gsSecondaryButton}
+                >
+                  -&gt; Explore more
+                </EAButton>
```

Comment:
I think this button stands out too much - I suggest just making it a "text button", in the same style as the "Sign up" button at the top right, and remove the arrow in it:

<img width="1437" alt="Screen Shot 2023-12-27 at 12 44 57 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/bef1cf15-92ae-4e35-aa09-70795e788ce4">